docs/design_decisions.md

```diff
@@ -108,6 +108,13 @@ Streef er naar het gebruik en interpretatie van gegevens in een resource (incl.
 _**Voorbeeld 1**_: I.p.v. voor de property 'indicatieGeheim' de waarden '0' of '1' te definiëren kun je ook de waarden 'true' en 'false' definiëren.
 _**Voorbeeld 2**_: I.p.v. de property 'inOnderzoek' kan je juist kiezen voor 'mogelijkOnjuist' waaruit veel duidelijker wordt wat de functie is.
 
+### DD1.19 Gebruik het schemacomponent xxxHalBasis als er gebruik wordt gemaakt van Hal
+
+Voor een Hal component die alleen bestaat uit _links voldoet de naam aan de volgende syntax:
```

Comment:
Wordt met deze rule gezegd dat je altijd een xxxHalBasis moet hebben? En dat je ook een xxxHal hebt die hiervan moet afleiden? Dat betekent dat je dan altijd een xxxHalBasis heb, ook als het nergens anders voor nodig is. Ik zou dus eerder redeneren dat je altijd xxxHal heb en als er ook behoefte is aan een variant zonder _embedded dat je dan pas xxxHalBasis introduceert.

Reply:
Deze discussie hebben we eerder gevoerd. Ik vind het uitstekend om het daar overnieuw over te hebben. Ik zal deze pr buiten v1.2.0. houden.

Reply:
Ik kan mij de discussie niet meer herinneren, sorry mijn fout.

Maar als mijn interpretatie klopt (dus altijd een HalBasis class waarvan de Hal class moet afleiden), dan moet in de toekomst BAG en BRK worden aangepast, omdat die dan niet hieraan conformeren.

Reply:
Ik vind het heel goed om componenten op een consequente manier te benoemen, maar als dit leidt een gevoel van "verplichting" tot wijzigingen aan BRK en BAG dan ben ik er wat minder enthousiast over. Deze naamgeving is namelijk alleen relevant voor de beheerder van de API schema's en niet voor de gebruikers (anders dan bij naamgeving van endpoints, parameters en properties). Hoort deze dan wel thuis in de design decisions?

Reply:
Ik kan me voorstellen dat je er niet enthousiast over wordt om de BRK en BAG API aan te moeten passen alleen om aan deze DD te voldoen. Als het echter op zich een goede DD is lijkt het me ook niet goed om hem te verwijderen. 
Wellicht voldoen deze API's dan misschien niet aan deze DD maar toekomstige toch zeker wel.
Ik ga er vanuit dat we nog wel meer DD's zullen opvoeren waaraan de API's die dan al bestaan niet zullen voldoen. Lijkt me ook geen reden om het DD document niet meer aan te passen. Het is immers ook een document waarin we ons voortschrijdend inzicht in vastleggen.